doc/rbd/rbd-mirroring.rst

```diff
@@ -228,6 +228,34 @@ For example::
    local cluster's ``rbd-mirror`` daemon process is responsible for performing
    the resync asynchronously.
 
+Mirror Status
+=============
+
+For every mirroring image the ``rbd-mirror`` daemon stores its current status
```

Comment:
"The peer cluster replication status is stored for every primary mirrored image. This status can be retrieved .."